big_tests/tests/cets_disco_SUITE.erl

```diff
@@ -123,15 +204,34 @@ mock_timestamp(Node, Timestamp) ->
     ok = rpc(Node, meck, new, [mongoose_rdbms_timestamp, [passthrough, no_link]]),
     ok = rpc(Node, meck, expect, [mongoose_rdbms_timestamp, select, 0, Timestamp]),
     %% Ensure that we mock
-    EnsureMocked = fun() ->
-        Timestamp = rpc(Node, mongoose_rdbms_timestamp, select, [])
-        end,
-    EnsureMocked(),
-    [{ensure_mocked, EnsureMocked}, {mocked_timestamp, Timestamp}].
-
-ensure_mocked(Config) ->
-    EnsureMocked = proplists:get_value(ensure_mocked, Config),
-    EnsureMocked().
+    Timestamp = rpc(Node, mongoose_rdbms_timestamp, select, []).
 
 unmock_timestamp(Node) ->
     ok = rpc(Node, meck, unload, [mongoose_rdbms_timestamp]).
+
+unmock(Node) ->
+    rpc(Node, meck, unload, []).
+
+random_cluster_name(CaseName) ->
+    Rand = rpc(mim(), mongoose_bin, gen_from_crypto, []),
+    <<"big_test_", (atom_to_binary(CaseName))/binary, "_", Rand/binary>>.
+
+insert_new(CN, BinNode, TS, NodeNum) ->
+    Ret = rpc(mim(), mongoose_cets_discovery_rdbms, insert_new, [CN, BinNode, TS, NodeNum]),
+    ct:pal("insert_new(~p, ~p, ~p, ~p) = ~p", [CN, BinNode, TS, NodeNum, Ret]),
```

Comment:
Use ct:log, if you still want to keep the log messages collected.
ct:log would put messages into the ct reports.
But not in the shell.
ct:pal has higher verbosity rate and only should be used to report when stuff goes wrong.